Makefile

```diff
@@ -62,7 +62,10 @@ test:
 
 vendor:
 	gem install bundler
+	bundle install
 
 bootstrap: Gemfile Gemfile.lock | vendor
-	bundle install
 	touch $@
+
+clean:
```

Comment:
This is nice. When do we think we'll want to run `make clean`? Another thing people should do now and again is `rm -rf _site`. If you don't you can see some weirdness in generated files (css, etc.). Should that be a part of `clean` as well?

Reply:
Good idea on removing `_site`. I'll push a new commit.

I think you'd run `make clean` whenever you need to reset your Jekyll environment. That could be because we updated the dependencies in `Gemfile`, Homebrew/rbenv updated the Ruby version, or because you're seeing an error when trying to build locally.